cmd/minikube/cmd/start.go

```diff
@@ -71,6 +72,7 @@ var (
 	insecureRegistry []string
 	apiServerNames   []string
 	apiServerIPs     []net.IP
+	hostRe           = regexp.MustCompile(`[\w\.-]+`)
```

Comment:
add a comment here

Reply:
Do you still want me to add the comment?